### ISSUE TITLE
App is not maintained: Reference alternative (Now Playing for Spotify)

### ISSUE THREAD

**Issue by paced** [citruspi/Spotify-Notifications#127]
TL;DR: Can the README.md file reference [Now Playing for Spotify](https://github.com/teaminkling/now-playing) as a viable alternative?

I should note this is somewhat self-promotional since I am the maintainer for [Now Playing for Spotify](https://github.com/teaminkling/mac-spotify-np).

Both this project (Spotify Notifications) and @davicorreiajr's [Spotify Now Playing](https://github.com/davicorreiajr/spotify-now-playing) are unmaintained (5 years and 2 years respectively as of writing).

I have focused on my hard fork of Spotify Now Playing and not Spotify Notifications but our apps overlap greatly.

Aside from continuing development, I have also copied over all of the pertinent issues from both repositories (including this one) and implemented a number of changes in a new version: v0.7.0 -> [v0.8.1](https://github.com/teaminkling/mac-spotify-np/releases/tag/v0.8.1).

What does the community think about referencing Now Playing for Spotify?

**Comment by tonglil** [comment]
FWIW to future readers, I want to say that **this app still works**.

I also like that it's mac-native using less than 50mb of memory (instead of wrapping nodejs in electron) which speaks to it's longevity and simplicity (does what it does).

However your alternative does have some additional features others may find useful.

**Comment by paced** [comment]
@tonglil Very fair points, especially on memory consumption. I've found numerous pain points while developing for [Now Playing for Spotify](https://github.com/teaminkling/now-playing) and it certainly needs to continuously be maintained.

The reason I started that project in the first place was because of the unanswered Issues on this one, though they might not all be relevant if the app largely works.

Your comment is timely as I no longer use Spotify and therefore [no longer dogfood my fork](https://github.com/teaminkling/now-playing/issues/114). I won't need to look for a new maintainer if this project works, though a few may miss the additional functionality. Thanks for the heads up!
